src/ansi-c/library/pthread_lib.c

```diff
@@ -293,10 +293,23 @@ inline int pthread_mutex_destroy(pthread_mutex_t *mutex)
 extern __CPROVER_bool __CPROVER_threads_exited[];
 extern __CPROVER_thread_local unsigned long __CPROVER_thread_id;
 
+extern __CPROVER_thread_local const void *__CPROVER_thread_keys[];
+extern __CPROVER_thread_local void (*__CPROVER_thread_key_dtors[])(void *);
+extern __CPROVER_thread_local unsigned long __CPROVER_next_thread_key;
+
 inline void pthread_exit(void *value_ptr)
 {
   __CPROVER_HIDE:;
   if(value_ptr!=0) (void)*(char*)value_ptr;
+#if 0
```

Comment:
Suggest to add also a comment in the code why this code has been deactivated.

Reply:
Done!